novuhq/website
# chore: hero command line to buttons

This pull request brings small update on the home page by replacing the command-line interface in the hero section with clickable buttons.

Steps to check:

- open [preview](https://deploy-preview-107--novu-website.netlify.app/)
- make sure that everything looks and feels as expected
